src/styles/main.scss

```diff
@@ -1,3 +1,60 @@
+@import "./variables.scss";
+
+* {
+  display: flex;
```

Comment:
```suggestion

```
It's better to apply this style directly

Reply:
It breaks your layout 
<img width="1082" alt="Screenshot 2023-11-20 at 14 25 47" src="https://github.com/mate-academy/layout_calendar/assets/135232001/90f46684-ba11-444c-adeb-d8677cd7a1f7">